src/components/InputDetail.tsx

```diff
@@ -5,23 +5,19 @@ import Tab from "@mui/material/Tab";
 import TextField from "@mui/material/TextField";
 import { Stack } from "@mui/system";
 import { useTheme } from "@mui/material";
+import { AddTask } from "./model";
 
-interface Props {
-  addTask: string;
-  setAddTask: React.Dispatch<React.SetStateAction<string>>;
-  handleAdd: (e: React.FormEvent) => void;
-}
 const frequencies = ["daily", "weekly", "monthly", "yearly"];
 
-const InputDetail: React.FC<Props> = ({ addTask, setAddTask, handleAdd }) => {
+const InputDetail: React.FC<AddTask> = ({ addTask, setAddTask, handleAdd }) => {
   const [value, setValue] = React.useState(0);
 
   let theme = useTheme();
 
   const handleChange = (event: React.SyntheticEvent, selectedIndex: number) => {
     setValue(selectedIndex);
     const frequency = frequencies[selectedIndex];
-    console.log({ frequency });
+    // console.log({ frequency });
   };
 
   return (
```

Comment:
you should have an `onChange` for each of these inputs.  You are passing in the functions to setTitle, setDescription, and setFrequency, so you'll want to use those in the `onChange`

for the title input, you should do something like:

```
onChange={(e) => {
    setTaskTitle(e.target.value)   
}}
```
and kinda do the same thing for the description input and the tabs onChange